aait/backend/group-2A/Application/Features/Like/Handlers/Queries/GetLikedPostRequestHandler.cs

```diff
@@ -0,0 +1,23 @@
+using Application.Contracts.Persistance;
+using Application.DTO.Post;
+using Application.Features.Like.Request.Queries;
+using AutoMapper;
+using MediatR;
+
+namespace Application.Features.Like.Handlers.Query;
+
+public class GetLikedPostRequestHandler : IRequestHandler<GetLikedPostRequest, List<PostDto>>{
+    private readonly IUnitOfWork _unitOfWork;
+    private readonly IMapper _mapper;
+
+    public GetLikedPostRequestHandler(IUnitOfWork unitOfWork, IMapper mapper){
+        _unitOfWork = unitOfWork;
+        _mapper = mapper;
+    }
+
+    public async Task<List<PostDto>> Handle(GetLikedPostRequest request, CancellationToken cancellationToken){
+        var posts = await _unitOfWork.likeRepository.GetLikedPost(request.Id);
```

Comment:
Check whether the posts list is null or empty before mapping it. null pointer exception will be thrown if you try to map a null.